coprocessor/fhevm-engine/fhevm-listener/src/cmd/mod.rs

```diff
@@ -272,6 +280,12 @@ impl InfiniteLogIter {
                 Err(err) => {
                     let delay = if not_initialized {
                         if retry == 0 {
+                            // TODO: remove panic and, instead, propagate the error
```

Comment:
Same here, at start after 20 retry failed we want a real failure. Error could be bubble up for display but it will end the service.

Reply:
Yeah, for sure, but better to not do it via a panic and log and exit with an error (not a panic).